tests/release/e2e-test-default-bundle.go

```diff
@@ -0,0 +1,249 @@
+	defer GinkgoRecover()
+	// Initialize the tests controllers
+	framework, err := framework.NewFramework()
+	Expect(err).NotTo(HaveOccurred())
+
+	var defaultEcPolicy = ecp.EnterpriseContractPolicySpec{}
+	var devNamespace = uuid.New().String()
+	var managedNamespace = uuid.New().String()
+
+	var cm = &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: buildPipelineBundleDefaultName,
+		},
+		Data: map[string]string{
+			"default_build_bundle": buildPipelineBundleDefault,
+		},
+	}
+
+	BeforeAll(func() {
+		kubeController := tekton.KubeController{
+			Commonctrl: *framework.CommonController,
+			Tektonctrl: *framework.TektonController,
+		}
+
+		_, err := framework.CommonController.CreateTestNamespace(devNamespace)
+		Expect(err).NotTo(HaveOccurred(), "Error when creating namespace: %v", err)
+		klog.Info("Dev Namespace created: ", devNamespace)
+
+		_, err = framework.CommonController.CreateTestNamespace(managedNamespace)
+		Expect(err).NotTo(HaveOccurred(), "Error when creating namespace: %v", err)
+		klog.Info("Managed Namespace created: ", managedNamespace)
+
+		klog.Infof("Wait until the 'pipeline' SA is created in %s namespace \n", managedNamespace)
+		Eventually(func() bool {
+			sa, err := framework.CommonController.GetServiceAccount("pipeline", managedNamespace)
+			return sa != nil && err == nil
+		}, 1*time.Minute, defaultInterval).Should(BeTrue(), "timed out when waiting for the \"pipeline\" SA to be created")
+
+		//utils.GetEnv(constants.QUAY_OAUTH_TOKEN_RELEASE_SOURCE, "")
+		sourceAuthJson := utils.GetEnv("QUAY_TOKEN", "")
+		Expect(sourceAuthJson).ToNot(BeEmpty())
+
+		destinationAuthJson := utils.GetEnv(constants.QUAY_OAUTH_TOKEN_RELEASE_DESTINATION, "")
+		Expect(destinationAuthJson).ToNot(BeEmpty())
+
+		_, err = framework.CommonController.CreateRegistryAuthSecret(hacbsReleaseTestsTokenSecret, devNamespace, sourceAuthJson)
+		Expect(err).ToNot(HaveOccurred())
+
+		_, err = framework.ReleaseController.CreateRegistryJsonSecret(redhatAppstudioUserSecret, managedNamespace, destinationAuthJson, destinationKeyName)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = framework.CommonController.LinkSecretToServiceAccount(devNamespace, hacbsReleaseTestsTokenSecret, "pipeline")
+		Expect(err).ToNot(HaveOccurred())
+
+		klog.Info("Dev Namespace:", devNamespace)
+		klog.Info("Managed Namespace:", managedNamespace)
+
+		// Copy the public key from tekton-chains/signing-secrets to a new
+		// secret that contains just the public key to ensure that access
+		// to password and private key are not needed.
+		publicKey, err := kubeController.GetPublicKey("signing-secrets", constants.TEKTON_CHAINS_NS)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(kubeController.CreateOrUpdateSigningSecret(
+			publicKey, publicSecretNameAuth, managedNamespace)).To(Succeed())
+
+		defaultEcPolicy = ecp.EnterpriseContractPolicySpec{
```

Comment:
why is this here and in `release.go`?

Reply:
the one in `release.go` is without label  **PublicKey**